### PR TITLE
Swope cnfc announce

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/cluster_lifecycle/import_intro.adoc [hunk 1]
```diff
@@ -3,9 +3,19 @@
...
 
-A hub cluster cannot manage _any_ other hub cluster, but can manage itself. The hub cluster is configured to automatically be imported and self-managed. You do not need to manually import the hub cluster. 
+* A hub cluster cannot manage _any_ other hub cluster, but can manage itself. The hub cluster is configured to automatically be imported and self-managed. You do not need to manually import the hub cluster. 
 
-If you remove a hub cluster and try to import it again, you must add the `local-cluster:true` label to the `ManagedCluster` resource.
+* If you remove a hub cluster and try to import it again, you must add the `local-cluster:true` label to the `ManagedCluster` resource.
+
+*Important:* Cluster lifecycle now supports all providers that are certified through the Cloud Native Computing Foundation (CNCF) Kubernetes Conformance Program. Choose a vendor that is recognized by CNFC for your hybrid cloud multicluster management.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/import_intro.adoc:10]: Did you want the Important note to be aligned with the second bullet point? If yes, I recommend adding the plus sign (+)
```suggestion
+
*Important:* Cluster lifecycle now supports all providers that are certified through the Cloud Native Computing Foundation (CNCF) Kubernetes Conformance Program. Choose a vendor that is recognized by CNFC for your hybrid cloud multicluster management. 
```

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -15,10 +26,13 @@ Access the link:https://access.redhat.com/articles/7055998[Support matrix] to le
 * <<dr4hub-whats-new,Backup and restore>>
 * <<net-whats-new,Networking>>
 
+Some features and components are identified and released as link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
+
+Access the link:https://access.redhat.com/articles/7055998[Support matrix] to learn about hub cluster and managed cluster requirements and support for each component.
+
 //[#installation]
 //== Installation no epics this release 2.10 bcs
```
Comment (dockerymick) [inline release_notes/whats_new.adoc:34]: are we comfortable with removing this comment since it was for 2.10?

**Review by dockerymick — APPROVED** [review]
Left a couple of suggestions and checked links. Thanks for adding this and working with PMs

File: clusters/cluster_lifecycle/import_intro.adoc (hunk 1 above)
Comment (swopebe) [inline clusters/cluster_lifecycle/import_intro.adoc:10]: No, this should stand alone as a paragraph, with the bullets that follow providing more detail.

File: release_notes/whats_new.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/whats_new.adoc:34]: @dockerymick When there is no topic, we have to comment it out. I am leaving only until I know I have nothing to add for 2.11, then I can change it, but good eye.

**Pull request merged**